crates/ruff/src/settings/types.rs

```diff
@@ -230,6 +230,10 @@ pub enum SerializationFormat {
     Gitlab,
     Pylint,
     Azure,
+    // We can't make this hidden in clap because clap is an optional dependency
+    // TODO(konstin): Remove ecosystem ci once [ruff::message::diff::Diff] is stable.
+    /// For use with check_ecosystem.py only
+    EcosystemCi,
```

Comment:
Is it possible to restrict this variant to a rust feature?

Reply:
done